include/picolibrary/wiznet/w5500.h

```diff
@@ -1282,6 +1282,12 @@ struct SN_SR {
 
     /**
      * \brief Status.
+     *
+     * \warning This enum does not cover all valid register values since there are
+     *          register values that are not documented, and these register values will
+     *          not be disclosed due to WIZnet company policy (see
+     *          https://forum.wiznet.io/t/topic/5020/2 and
+     *          https://forum.wiznet.io/t/topic/8883/2).
```

Comment:
```suggestion
     * \warning This enum does not cover all valid register values since there are
     *          register values that are not documented in the datasheet, and these
     *          register values will not be disclosed due to WIZnet company policy (see
     *          https://forum.wiznet.io/t/topic/5020/2 and
     *          https://forum.wiznet.io/t/topic/8883/2).
```